cloud/pkg/cloudhub/handler/messagehandler.go

```diff
@@ -140,8 +140,13 @@ func (mh *MessageHandle) OnRegister(connection conn.Connection) {
 	nodeID := connection.ConnectionState().Headers.Get("node_id")
 	projectID := connection.ConnectionState().Headers.Get("project_id")
 
+	nodeID = nodeID + "-" + connection.RemoteAddr().String()
```

Comment:
Why do we need add `connection.RemoteAddr().String()`  here？

Reply:
The difference between connection 1 and connection 2 is the remote port. In order to distinguish the two connected channels, the original nodeid + remote addr (ip: port) is used as the channel name.